src/Desktop.Analyzers/UnitTests/DoNotUseInsecureDTDProcessingUseXmlReaderForSchemaReadTests.cs

```diff
@@ -0,0 +1,410 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.UnitTests;
+using Microsoft.CodeAnalysis.CSharp;
+using Xunit;
+
+namespace Desktop.Analyzers.UnitTests
+{
+    public partial class DoNotUseInsecureDTDProcessingAnalyzerTests : DiagnosticAnalyzerTestBase
+    {
+        private DiagnosticResult GetCA3075CSharpResultAt(int line, int column, string name)
```

Comment:
In the other "insecure DTD processing" files, you used more specific names, such as `GetCA3075XPathDocumentCSharpResultAt`. I assume you didn't use the more specific name here because you wrote this file first. I suggest `GetCA3075SchemaReadCSharpResultAt`.